db/seeds.rb

```diff
@@ -1,7 +1,11 @@
+# frozen_string_literal: true
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+Role.create(name: 'User')
```

Comment:
try `Role.create_all([{ name: 'User', name: 'Admin' }])`
Not important here, but usually in controllers or other places where response time is important, it's better to hit the database lesser times for performance.